packages/go_router_builder/pubspec.yaml

```diff
@@ -18,7 +18,9 @@ dependencies:
   meta: ^1.7.0
   path: ^1.8.0
   path_to_regexp: ^0.4.0
-  source_gen: ^1.0.0
+  #TODO(tarrinneal): Unpin once source_gen_test is updated to fix collection issue.
+  
+  source_gen: 1.2.7
```

Comment:
Pinning in the actual package would affect clients; since we only need it for dev we should put this in a `dependency_overrides:` section instead.